AMW_angular/io/src/app/settings/settings.component.html

```diff
@@ -12,7 +12,7 @@
         <a class="nav-link" href="/AMW_web/pages/settings.xhtml?mode=STP">STP Management</a>
         <a class="nav-link" href="/AMW_web/pages/settings.xhtml?mode=releases">Releases</a>
         <a class="nav-link" routerLink="/settings/permission" routerLinkActive="active">Roles and Permissions</a>
-        <a class="nav-link" href="/AMW_web/pages/settings.xhtml?mode=appVersion">Application Info</a>
+        <a class="nav-link" routerLink="/settings/application-info" routerLinkActive="active">Application Info</a>
       </nav>
     </div>
     <div class="col-9">
```

Comment:
Can we go to `col-10` here? With `cole-9` the page has unused extra space on the right:
![image](https://github.com/liimaorg/liima/assets/15231595/9e88b4b9-cfb1-48cd-9310-d5d2967f0dff)
This page could benefit from some more space and this size is also in line with the other pages (deploy and audit).

Reply:
yes, makes sense